utils/shortcodes.js

```diff
@@ -1,5 +1,15 @@
-const Image = require("@11ty/eleventy-img");
-const { cacheBuster } = require("../src/_data/globals");
+const Image = require('@11ty/eleventy-img');
+const { cacheBuster } = require('../src/_data/globals');
+const marked = require('marked');
+
+function toHtml(markdown) {
+  let rendered = marked
```

Comment:
I just had to tweak this function to prevent a crash on the server:

```suggestion
  if (!markdown) return '';
 
  let rendered = marked
```